Source/Core/DolphinQt/Config/Mapping/MappingNumeric.cpp

```diff
@@ -41,6 +37,35 @@ void MappingDouble::fixup(QString& input) const
 void MappingDouble::ConfigChanged()
 {
   const QSignalBlocker blocker(this);
+
+  QString suffix;
+
+  if (const auto ui_suffix = m_setting.GetUISuffix())
+    suffix += QLatin1Char{' '} + tr(ui_suffix);
+
+  if (m_setting.IsSimpleValue())
+  {
+    setRange(m_setting.GetMinValue(), m_setting.GetMaxValue());
+    setButtonSymbols(ButtonSymbols::UpDownArrows);
+  }
+  else
+  {
+    setRange(-std::numeric_limits<double>::infinity(), std::numeric_limits<double>::infinity());
```

Comment:
I was wondering if there was `negative_infinity` anywhere (spoilers: there isn't; and using `-infinity` is the way to go), but while looking that up I noticed that you're missing an `#include <limits>` in this file.